dll/win32/msafd/misc/sndrcv.c

```diff
@@ -166,9 +196,9 @@ WSPRecv(SOCKET Handle,
 {
     PIO_STATUS_BLOCK        IOSB;
     IO_STATUS_BLOCK         DummyIOSB;
-    AFD_RECV_INFO           RecvInfo;
+    PAFD_RECV_INFO          RecvInfo;
```

Comment:
Why switching from a local variable to a heap-allocated variable that gets freed at the end of the function ?